changelogs/fragments/package_facts_warnings.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - package_facts - returns the correct warning when package listing fails.
+  - package_facts - no longer fails silently when the selected package manager is unable to list packages.
```

Comment:
```suggestion
  - package_facts - no longer fails silently when the selected package manager is unable to list packages (https://github.com/ansible/ansible/issues/83502).
```

Reply:
Sorry, thought I'd link the issue in the porting guide quickly before merging but CI had other ideas. Closing/reopening to fix it :crossed_fingers:

Reply:
Reverted it